Makefile.prow

```diff
@@ -1,92 +1,90 @@
-# warning: verify this inside a container with following:
-# docker run -it --rm -v `pwd`:`pwd` -w `pwd` golang:1.18 bash -c "make -f Makefile.prow ci"
-# 
-# otherwise unintended system changes may include, replacing your currently installed kubectl, binaries in GOBIN, files in GOPATH/src
-# 
-# If formatting needs updating you can run
+# Warning: Verify this inside a container with the following command:
+# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-v1.22.2 bash -c "make -f Makefile.prow ci"
+# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-v1.22.2 bash -c "make -f Makefile.prow lint"
```

Comment:
Actually do we wanna use ubi9-latest here?
https://github.com/openshift/velero/blob/oadp-1.4/Dockerfile.ubi#L1

Reply:
```suggestion
# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-latest bash -c "make -f Makefile.prow ci"
# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` quay.io/konveyor/builder:ubi9-latest bash -c "make -f Makefile.prow lint"
```

Reply:
will just leave as is.. for now.